modules/firebase/firebase_test.go

```diff
@@ -0,0 +1,47 @@
+package firebase_test
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/testcontainers/testcontainers-go/modules/firebase"
+)
+
+func TestFirebase(t *testing.T) {
+	ctx, cancel := context.WithDeadline(context.Background(), time.Now().Add(3*time.Minute))
+
+	thing := fmt.Sprintf("%s/test", os.Getenv("PWD"))
+	fmt.Println(thing)
+
+	container, err := firebase.RunContainer(
+		ctx,
+		firebase.WithRoot(filepath.Join(BasePath(), "firebase")),
+		firebase.WithCache(),
+	)
+	if err != nil {
```

Comment:
bug: container can require clean up see other modules and template for how to handle this. We're also standarising on require for checks like this in tests.